plugins/autosuspend.koplugin/main.lua

```diff
@@ -192,6 +194,7 @@ function AutoSuspend:init()
         self:_schedule(shutdown_only)
     end
     if Device:isKindle() then
+        self.last_t1_reset_time = UIManager:getElapsedTimeSinceBoot()
```

Comment:
Shouldn't that initialize to 0 lest the device enter standby within mere seconds after starting the program?

Reply:
Sure, makes sense.